src/common/components/signInModal/signInModal.tpl.html

```diff
@@ -5,7 +5,7 @@ <h4 class="text-center" translate>If you don&apos;t have a Cru username, you can
     <hr class="horizontal-divider"/>
     <h4 class="text-center" translate>If you already have a Cru username, use it to sign in so we can link it to your giving</h4>
   </div>
-  <p ng-if="$ctrl.identified" class="text-center">
+  <p ng-if="typeof $ctrl.sessionService.session.first_name != 'undefined' && $ctrl.sessionService.session.first_name && typeof $ctrl.sessionService.session.last_name != 'undefined' && $ctrl.sessionService.session.last_name" class="text-center">
```

Comment:
```suggestion
  <p ng-if="$ctrl.identified && $ctrl.sessionService.session.first_name && $ctrl.sessionService.session.last_name" class="text-center">
```

Reply:
Identified is the only time we will have their name?  What is "Registered" again?

Reply:
Registered is if they are fully logged in. Identified is the only time they will need to login and we have their name.